GitUI/Editor/FileViewer.cs

```diff
@@ -617,7 +617,7 @@ public static string ToHexDump(byte[] bytes, StringBuilder str, int columnWidth
 
             var i = 0;
 
-            while (i < bytes.Length)
+            while (i < bytes.Length && i < maxBytes)
```

Comment:
```suggestion
            var limit = Math.Min(bytes.Length, maxBytes);
            while (i < limit)
```

Reply:
Will change
(I did implement like this at first, but reverted to min number of lines. )